### PR TITLE
Black 20 is the new black

### PR DESCRIPTION
**Pull request by euri10**
black handles things a little bit differently in v20

### CODE REVIEW

File: tests/protocols/test_http.py [hunk 1]
```diff
@@ -696,7 +696,10 @@ async def asgi(receive, send):
 @pytest.mark.parametrize("asgi2or3_app, expected_scopes", asgi_scope_data)
 @pytest.mark.parametrize("protocol_cls", HTTP_PROTOCOLS)
 def test_scopes(asgi2or3_app, expected_scopes, protocol_cls):
-    protocol = get_connected_protocol(asgi2or3_app, protocol_cls,)
+    protocol = get_connected_protocol(
+        asgi2or3_app,
+        protocol_cls,
+    )
```
Comment (JayH5) [inline tests/protocols/test_http.py]:
```suggestion
    protocol = get_connected_protocol(asgi2or3_app, protocol_cls)
```

File: tests/test_config.py [hunk 2]
```diff
@@ -127,7 +127,10 @@ def test_log_config_default(mocked_logging_config_module, use_colors, expected):
 
     mocked_logging_config_module.dictConfig.assert_called_once_with(LOGGING_CONFIG)
 
-    ((provided_dict_config,), _,) = mocked_logging_config_module.dictConfig.call_args
+    (
+        (provided_dict_config,),
+        _,
+    ) = mocked_logging_config_module.dictConfig.call_args
```
Comment (JayH5) [inline tests/test_config.py]:
```suggestion
     (provided_dict_config,), _ = mocked_logging_config_module.dictConfig.call_args
```

**Review by JayH5 — COMMENTED** [review]
The issue here is that black now treats trailing newlines as "make this multi-line". The better solution is just to remove trailing newlines if we don't need them.

**Comment by euri10** [comment]
TIL I learnt trailing comma black subtlety ! should be ok now @JayH5

**Review by JayH5 — APPROVED** [review]

**Comment by lovelydinosaur** [comment]
Best issue title of the day.
Black *is* the new black. 🖤

**Pull request merged**